components/MenueHeader.js

```diff
@@ -1,53 +1,62 @@
 import Image from "next/image";
 import styled from "styled-components";
-import usedMenue from "../public/images/UsedMenue.svg";
-import menue from "../public/images/Menue.svg";
+import map from "../public/images/map.svg";
 import userIcon from "../public/images/userIcon.svg";
 import usedUserIcon from "../public/images/usedUserIcon.svg";
+import list from "../public/images/list.svg";
+import usedList from "../public/images/usedList.svg";
 
 export default function MenueHeader({
   onMapShown,
   onUserShown,
   setUserIcon,
-  setMenuIcon,
+  onListShown,
+  setListIcon,
 }) {
   return (
     <>
       <StyledHeader>
-        {setUserIcon ? (
+        {setListIcon ? (
           <Image
-            onClick={onUserShown}
-            src={usedUserIcon}
-            alt="Button um auf die Profilseite zu gelangen"
+            onClick={onListShown}
+            src={usedList}
+            alt="Button um auf die List seite zu kommen"
```

Comment:
The alt tag should only contain what you see, not what the functionality is. 
If it is a button, it should be a button. Alternatively, you could use some ARIA-roles, that lets your image appear as a button for supporting devices.

Reply:
same for the next images ;)